GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -462,6 +463,11 @@ void onDrawEyeView(int eye, float fovDegrees) {
         if (!(mSensoredScene == null || !mMainScene.equals(mSensoredScene))) {
             GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
 
+            if(eye == 0) {
+                GVRPerspectiveCamera centerCamera = mainCameraRig.getCenterCamera();
+                cull(mMainScene.getNative(), centerCamera.getNative(), mRenderBundle.getMaterialShaderManager().getNative());
+            }
+
```

Comment:
By general design, I am thinking onDrawEyeView, which will be called twice per frame, will handle rendering for each eye. And onDrawFrame is for pre-draw processing for a frame. I am just wondering if there is any reason that you put the culling here but not onDrawFrame?